components/list/list-item-placement-marker.js

```diff
@@ -0,0 +1,49 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class ListItemPlacementMarker extends LitElement {
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+				position: relative;
+			}
+
+			:host([hidden]) {
+				display: none;
+			}
+
+			svg {
+				width: 100%;
+			}
+
+			circle {
+				stroke: var(--d2l-color-celestine);
+				fill: white;
+			}
+
+			line {
+				stroke: var(--d2l-color-celestine);
+				stroke-linecap: round;
+			}
+		`;
+	}
+
+	constructor() {
+		super();
+		this.strokeWidth = 4;
+		this.radius = this.strokeWidth * 2;
+		this.height = this.radius * 2 + this.strokeWidth * 2;
+	}
+
+	render() {
```

Comment:
I would define height, radius and strokewidth within the render method over the constructor. I don't see reason to have their scope extended to the class.

Reply:
The advantage of the constructor would be that users can alter these properties if they need to, but I'd argue in that case we should have a single property such as `size` that does all of that.

Reply:
Unless we have a use case for a user to modify the size I dont think we should support it. Or if we do have it modified by css height not a var